docs/DeveloperGuide.adoc

```diff
@@ -89,7 +93,7 @@ The `UI` component,
 * Listens for changes to `Model` data so that the UI can be updated with the modified data.
 
 [[Design-Logic]]
-=== Logic component
+=== Logic Component
 
 [[fig-LogicClassDiagram]]
```

Comment:
Have you forgotten to change the name of AddressBookParser to ModManagerParser in the Logic class diagram?